ecommerce/frontend/src/components/pages/LoginPage.less

```diff
@@ -0,0 +1,34 @@
+.loginForm {
```

Comment:
Class names are written with **hyphen-separated-strings** style. It makes it easy to read. 
A better example would be -> login-form / login-buttons

Reply:
agree, let's stick to conventions.